tornettools/parse_tgen.py

```diff
@@ -97,27 +97,21 @@ def __extract_client_goodput_5MiB(args, data, circuittype, startts, stopts):
 def __get_download_time(data, circuittype, startts, stopts, bytekey):
     dt = {'ALL': []}
 
-    # download times can differ by microseconds in tgen
-    resolution = 1.0 / 1000000.0
-
     pattern = re.compile(r'perfclient\d+' + circuittype)
     if 'data' in data:
         for name in data['data']:
             if pattern.match(name) is None:
                 continue
             db = data['data'][name]
             ss = db['tgen']['stream_summary']
-            mybytes, mytime = 0, 0.0
             if bytekey in ss:
                 for header in ss[bytekey]:
-                    bytes = int(header)
                     for secstr in ss[bytekey][header]:
                         sec = int(secstr) - 946684800
                         if sec >= startts and (stopts < 0 or sec < stopts):
                             #mydlcount += len(data['nodes'][name]['lastbyte'][header][secstr])
                             for dl in ss[bytekey][header][secstr]:
                                 seconds = float(dl)
-                                #item = [seconds, resolution]
```

Comment:
This line shows _how_ we should incorporate the resolution error. Basically, if we pass in a single value, it computes the shaded CI from the value. But if we pass in a 2-tuple, then it uses the first as the value and the second as the resolution for the purposes of calculating resolution error.

My guess is that this was causing an error of some sort and so got commented, and then never fixed.

We should keep it commented here with a TODO.

Reply:
Done